src/applications/vaos/reducers/expressCare.js

```diff
@@ -46,34 +47,42 @@ function setupFormData(data, schema, uiSchema) {
 export default function expressCareReducer(state = initialState, action) {
   switch (action.type) {
     case FORM_PAGE_OPENED: {
+      const newRequest = { ...state.newRequest };
```

Comment:
I don't think you need to shallow copy this here. You're not modifying `newRequest` directly anywhere.